### PR TITLE
Update deep_svdd.py

### PR DESCRIPTION
**Pull request by modanesh**
Resolving `loss.backward()` error. It resolves issue #606 

### All Submissions Basics:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Have you checked all [Issues](../../issues) to tie the PR to a specific one?

### All Submissions Cores:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [ ] Does your submission pass tests, including CircleCI, Travis CI, and AppVeyor?
* [ ] Does your submission have appropriate code coverage? The cutoff threshold is 95% by Coversall.

### New Model Submissions:

* [ ] Have you created a <NewModel>.py in ~/pyod/models/?
* [ ] Have you created a <NewModel>_example.py in ~/examples/?
* [ ] Have you created a test_<NewModel>.py in ~/pyod/test/?
* [ ] Have you lint your code locally prior to submission?

### CODE REVIEW

**Comment by yzhao062** [comment]
thanks for the PR -- also by any chance could you redirect it to the development branch? thanks, Yue

**Comment by modanesh** [comment]
@yzhao062 thanks for the review. I just redirected my PR to the development branch.
